### PR TITLE
fix: Show error details on deleteEvent()

### LINKED ISSUE #3508: Proper error message should be displayed when deleting an event
**Describe the bug**
When deleting an event associated with orders, it should show the correct error message as it gets from the server.

**Additional context**
On it.

### PR DESCRIPTION
**Pull request by roshnet**
Checks if the variable is an array, and shows the error details if present.

Fixes #3508

#### Short description of what this resolves:

This shows an error message (if any) instead of the default error message string.

#### Checklist

- [X] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [X] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by roshnet** [comment]
Requesting review, @iamareebjamal / @prateekj117 .

File: app/controllers/admin/events/list.js [hunk 1]
```diff
@@ -132,7 +132,15 @@ export default class extends Controller.extend(EmberTableControllerMixin) {
       this.notify.success(this.l10n.t('Event has been deleted successfully.'));
     } catch (e) {
       console.warn(e);
-      this.notify.error(this.l10n.t('An unexpected error has occurred.'));
+      if (e.constructor === Array && 'errors' in e) {
```
Comment (iamareebjamal) [inline app/controllers/admin/events/list.js]: Hmm, For an array it'll be true, but `'errors' in e` will be false

For `{ errors: [] }`, vice versa, so it'll always resolve to false?

**Comment by uds5501** [comment]
@roshnet any updates?

**Comment by roshnet** [comment]
@uds5501 @iamareebjamal hi, sorry for the inactivity. I'll work on the reviews in a couple days.

**Comment by kushthedude** [comment]
Stale ?
Please feel free to reopen when you have worked on the reviews. Thanks.

**Pull request closed without merging**